api/bibliotheca.py

```diff
@@ -733,7 +733,7 @@ def create_default_start_time(self, _db, cli_date):
 
         The command line date argument should have the format YYYY-MM-DD.
         """
-        initialized = get_one(_db, Timestamp, self.service_name)
+        initialized = get_one(_db, Timestamp, service=self.service_name)
```

Comment:
This ancient bug was uncovered by the fact that the Bibliotheca Timestamp is no longer the first one in the database when this code runs during `TestBibliothecaEventMonitor.test_default_start_time`.